Sources/tbDEX/Protocol/DevTools.swift

```diff
@@ -0,0 +1,252 @@
+import Foundation
+import TypeID
+
+/// DevTools provides a few utility helper methods for constructing Resources and Messages.
+/// Should be used for internal testing purposes only.
+enum DevTools {
+    /// Creates a mock `Offering`. Optionally override the `OfferingData`
+    /// - Parameters:
+    ///   - from: The DID that the `Offering` should be from. Included in the metadata
+    ///   - data: Optional. The data to override the default data with.
+    ///   - protocol: Optional. The protocol version to use if different from the default. Included in the metadata.
+    /// - Returns: The `Offering`
+    static func createOffering(
+        from: String,
+        data: OfferingData? = nil,
+        protocol: String? = nil
+    ) -> Offering {
+        let offeringData = data ?? OfferingData(
+            description: "test offering",
+            payoutUnitsPerPayinUnit: "1",
+            payinCurrency: .init(currencyCode: "AUD"),
```

Comment:
i feel like this now lives in `payin` ?

Reply:
from https://github.com/TBD54566975/tbdex/blob/main/specs/protocol/README.md#offering: